src/imitation/scripts/common/train.py

```diff
@@ -98,6 +104,26 @@ def eval_policy(
     return rollout.rollout_stats(trajs)
 
 
+@train_ingredient.capture
+def save_video(
```

Comment:
When you call this function it self-documents as if the video were always saved. (but a flag indicating whether this should happen is magically injected through a decorator). I don't have an immediately better alternative, but perhaps a more explanatory function name could help.